### PR TITLE
[CN-1214] Introduce page for air-gapped env installation

### PR DESCRIPTION
**Pull request by hasancelik**

### CODE REVIEW

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
+= Running in air-gapped environments
+
+The Hazelcast Platform Operator can be run in an air-gapped environment without access to the open internet when it is configured not to pull container images from `docker.io/hazelcast`. 
+
+By default the Platform Operator, Hazelcast, the agent and Management Center image do not require you to specify the container images. The operator will automatically set the correct container image for each applications. When running in an air-gapped or offline environment you will have to mirror the official images in a private container image registry. To make use of your mirrored images you have to set the image for each applications explicitly as shown in the preceding sections.
+
+== Setting image for the Platform Operator
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:7]:
```suggestion
== Setting the image for the Platform Operator
```

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+----
+helm install operator hazelcast/hazelcast-platform-operator --version=5.13.0 \
+    --set=installCRDs=true \
+    --set=image.repository= < PRIVATE_REGISTRY > \
+    --set=image.tag= < PRIVATE_TAG > \
+    --set=image.pullSecrets= {< REGISTRY-SECRET >}
+----
+
+
+== Setting image for the Hazelcast
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:19]:
```suggestion
== Setting the image for Hazelcast
```

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+metadata:
+  name: hazelcast
+spec:
+  repository: < PRIVATE_REGISTRY >
+  version: < PRIVATE_TAG >
+  imagePullSecrets:
+    - name: < PRIVATE_REGISTRY-SECRET >
+```
+
+== Setting image for the Sidecar Agent
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:34]:
```suggestion
== Setting the image for the Sidecar Agent
```

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  name: hazelcast
+spec:
+  imagePullSecrets:
+    - name: < PRIVATE_REGISTRY-SECRET >
+  agent:
+    repository: < PRIVATE_REGISTRY >
+    version: < PRIVATE_TAG >
+```
+
+== Setting image for the Management Center
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:50]:
```suggestion
== Setting the image for the Management Center
```

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+metadata:
+  name: managementcenter
+spec:
+  repository: < PRIVATE_REGISTRY >
+  version: < PRIVATE_TAG >
+  imagePullSecrets:
+    - name: < PRIVATE_REGISTRY-SECRET >
+```
+
+== Setting image for the Flow
```
Comment (amandalindsay) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:65]:
```suggestion
== Setting the image for Flow
```

**Review by amandalindsay — CHANGES_REQUESTED** [review]
Just some minor docs tweaks and a question (I'm unfamiliar with the content) otherwise looks good

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 6]
```diff
@@ -0,0 +1,78 @@
+= Running in air-gapped environments
+
+The Hazelcast Platform Operator can be run in an air-gapped environment without access to the open internet when it is configured not to pull container images from `docker.io/hazelcast`. 
+
+By default, the Platform Operator, Hazelcast, the agent and Management Center image do not require you to specify the container images. The operator will automatically set the correct container image for each application. When running in an air-gapped or offline environment, you will have to mirror the official images in a private container image registry. To make use of your mirrored images, you have to set the image for each application explicitly, as shown in the preceding sections.
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:5]:
```suggestion
By default, while using the Hazelcast Platform Operator, you don't need to specify the container images in Hazelcast and Management Center custom resources for any application. The operator automatically sets the correct container image for each application. When running in an air-gapped or offline environment, you have to mirror the official images in a private container image registry. To make use of your mirrored images, you have to set the image for each application explicitly, as shown in the preceding sections.
```

File: docs/modules/ROOT/pages/air-gapped-env.adoc [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+== Setting the image for the Platform Operator
+
+[source,bash]
+----
+helm install operator hazelcast/hazelcast-platform-operator --version=5.13.0 \
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/pages/air-gapped-env.adoc:11]:
```suggestion
helm install operator hazelcast/hazelcast-platform-operator --version={operator-chart-version} \
```

**Review by kutluhanmetin — APPROVED** [review]

**Review by amandalindsay — APPROVED** [review]

**Pull request merged**